src/Script/RamCalculations.ts

```diff
@@ -92,7 +101,7 @@ function parseOnlyRamCalculate(otherScripts: Map<ScriptFilePath, Script>, code:
   }
 
   // Parse the initial module, which is the "main" script that is being run
-  const initialModule = "__SPECIAL_INITIAL_MODULE__";
```

Comment:
There's some comment references to this that aren't fixed up.

Reply:
changed it to <name of the main script> gonna push it when the other stuff is resolved